biorbd_optim/optimal_control_program.py

```diff
@@ -497,20 +497,31 @@ def solve(self, solver="ipopt", show_online_optim=False, options_ipopt={}):
         # Solve the problem
         return solver.call(arg)
 
-    def save(self, sol, file_path, to_numpy=False, **parameters):
+    def save(self, sol, file_path, to_numpy=(), **parameters):
```

Comment:
Is there a reason not to use Bool here? I think it makes it less verbose...

Reply:
Ok, I see why.. I think we should simplify the following code. This should be False be default and check the given extension accordingly. If no extension and False => .bo, if no extension and True => .bob (and inverse). If the True or False is incompatible with a provided extension it raises